### PR TITLE
Update versions in README.md

### PR DESCRIPTION
**Pull request by TWiStErRob**
and fix typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,20 +64,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: suzuki-shunsuke/github-action-renovate-config-validator@v0.1.2
+      - uses: suzuki-shunsuke/github-action-renovate-config-validator@v1
```
Comment (suzuki-shunsuke) [inline README.md:67]:
```suggestion
      - uses: suzuki-shunsuke/github-action-renovate-config-validator@v1.0.1
```

There is no tag `v1`.

File: README.md [hunk 2]
```diff
@@ -64,20 +64,20 @@ jobs:
...
 ```
 
 You can specify renovate-config-validator version and configuration file path.
 
 ```yaml
 steps:
-  - uses: suzuki-shunsuke/github-action-renovate-config-validator@v0.1.2
+  - uses: suzuki-shunsuke/github-action-renovate-config-validator@v1
```
Comment (suzuki-shunsuke) [inline README.md:74]:
```suggestion
  - uses: suzuki-shunsuke/github-action-renovate-config-validator@v1.0.1
```

**Review by suzuki-shunsuke — CHANGES_REQUESTED** [review]

**Comment by suzuki-shunsuke** [comment]
Thank you!
Nice catch!

**Comment by TWiStErRob** [comment]
There should be a v1 following the minor/patch releases!

**Comment by TWiStErRob** [comment]
Docs: https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management (point 3)

File: README.md (hunk 1 above)
Comment (TWiStErRob) [reply README.md:67]: There should be a v1 following the minor/patch releases!

Docs: https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management (point 3)

**Comment by suzuki-shunsuke** [comment]
I don't like the practice. I believe we should specify the full semantic version strictly.
So I never create tags such as `v1`.

**Comment by TWiStErRob** [comment]
I believe you should let your users decide which pattern they use. It's surprising when an action doesn't have this. I also agree and encourage pinning like [this](https://github.com/detekt/detekt/blob/main/.github/workflows/fossascan.yaml#L16-L20), but there are many many projects where that's not set up, or not necessary.

**Comment by TWiStErRob** [comment]
Anyway, up to you. I'm not personally using this action (and wouldn't even if I need to, because of the above). I was just fixing Glide's renovate config.

**Comment by TWiStErRob** [comment]
Feel free to accept or reject your suggestions and merge this.

**Pull request merged**